MainApp/MainApp/Coordinator/Feature/Profile/EditProfileCoordinator.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  EditProfileCoordinator.swift
+//  MainApp
+//
+//  Created by Owen Prasetya on 03/08/20.
+//  Copyright © 2020 William Inx. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class EditProfileCoordinator: EditProfileCoordinatorProtocol {
```

Comment:
please combine editProfileCoordinator and profileLandingCoordinator as it is one same feature. name it ProfileCoordinator

Reply:
Combined the coordinator and protocol